Solutions/SlashNext SIEM/Playbook/deploy.json

```diff
@@ -0,0 +1,1192 @@
+                                                                    "loggingaction": {
+                                                                        "type": "string",
+                                                                        "description": "loggingaction"
+                                                                    },
+                                                                    "eptypeText": {
+                                                                        "type": "string",
+                                                                        "description": "eptypeText"
+                                                                    },
+                                                                    "phishingUrl": {
+                                                                        "type": "string",
+                                                                        "description": "phishingUrl"
+                                                                    },
+                                                                    "infectionvector": {
+                                                                        "type": "string",
+                                                                        "description": "infectionvector"
+                                                                    },
+                                                                    "threattype": {
+                                                                        "type": "string",
+                                                                        "description": "threattype"
+                                                                    }
+                                                                }
+                                                            }
+                                                        }
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "definitions": {
+                    },
+                    "parameters": {
+                    },
+                    "responses": {
+                    },
+                    "securityDefinitions": {
+                        "api_key": {
+                            "type": "apiKey",
+                            "in": "header",
+                            "name": "apikey"
+                        }
+                    },
+                    "security": [
+                    ],
+                    "tags": [
+                    ]
+                }
+            }
+        },
+        {
+            "type": "Microsoft.Storage/storageAccounts",
```

Comment:
Our current solution packaging doesn't support creation of active resources like storage accounts,

Please consider use of existing storage account in playbook and update readme file by asking customer to deploy storage account manually and then use it  while deploying playbook

Reply:
Active resources are excluded.